src/core/components/object.js

```diff
@@ -65,6 +65,22 @@ function parseProtoBuffer (buf, callback) {
   dagPB.util.deserialize(buf, callback)
 }
 
+// Recursively search the node for CIDs
+function getNodeLinks (node, path = '') {
+  let links = []
+  for (const [name, value] of Object.entries(node)) {
+    if (CID.isCID(value)) {
+      links.push({
+        name: path + name,
+        cid: value
+      })
+    } else if (typeof value === 'object') {
+      links = links.concat(getNodeLinks(value, path + name + '/'))
+    }
+  }
+  return links
+}
```

Comment:
Ah, I see, we need to leave the object API alone - it's considered deprecated and will be removed in a future release. We should use the DAG api for this or IPLD directly.

Reply:
@alanshaw should I revert this commit then?
https://github.com/ipfs/interface-js-ipfs-core/pull/460/commits/ba4f788978d5a2767ff5c41272541ee6be8b2d2e

Reply:
Created PR to revert the test here: https://github.com/ipfs/interface-js-ipfs-core/pull/465